### DISCUSSION TITLE
What is a callback function in JavaScript?

### DISCUSSION THREAD

**Discussion by ajay-dhangar** [JavaScript-Mastery/javascript-mastery#16] [Q&A]
A callback function in JavaScript is a function that is passed as an argument to another function, and is executed when the parent function is called. Callback functions are often used in asynchronous programming, where you need to wait for a response before continuing with the code.
**Example:**

```js
function myFunction(callback) {
  // do something
  callback();
}

myFunction(function() {
  console.log("callback function executed");
});
```

**Comment by Bruno-Luna** [comment]
Yes, very good!
https://developer.mozilla.org/en-US/docs/Glossary/Callback_function
